ImageItem/ImageItem.js

```diff
@@ -43,6 +43,14 @@ const ImageItemBase = kind({
 		 */
 		children: PropTypes.node,
 
+		/**
+		 * Display caption over image.
+		 *
+		 * @type {String}
+		 * @public
+		 */
+		childrenOverImage: PropTypes.bool,
```

Comment:
maybe it is better to rename the prop "captionOverImage" ? I saw you used  this also as a class name